html/js/product-multilingual.js

```diff
@@ -211,10 +215,10 @@ function rotate_image(event) {
     angles[imagefield] += angle;
     angles[imagefield] = (360 + angles[imagefield]) % 360;
 
-    $('img#crop_' + imagefield).cropper('rotate', angle);
+    croppers[imagefield]?.rotate(angle);
```

Comment:
It could be better to store the current cropper in a variable and exit early if it's `null`/`undefined`. Would make following calls easier by not having to use the null-forgiving operator.
Also, some later calls like `croppers[imagefield]?.getCropBoxData();` would not return anything usable, but aren't guarded against the cropper not existing for some reason